FlightPlanner.Web/FlightPlanner.Web/Storage/FlightStorage.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FlightPlanner.Web.Interface;
+using FlightPlanner.Web.Models;
+
+namespace FlightPlanner.Web.Storage
+{
+    public static class FlightStorage
+    {
+        private static readonly List<Flight> _flights = new();
+
+        private static readonly AirportStorage _airportStorage = new();
+        private static int _id;
+
+        public static Flight GetFlight(int id)
+        {
+            return FindFlight(id);
+        }
+
+        public static Flight FindFlight(int id)
+        {
+            return _flights.SingleOrDefault(f => f.Id == id);
+        }
+
+        public static void ClearFlights()
+        {
+            _flights.Clear();
+        }
+
+        public static Flight PutFlight(AddFlightRequest flightRequest)
+        {
+            List<Airport> airports = new List<Airport>
+            {
+                flightRequest.From,
+                flightRequest.To
+            };
+
+            _airportStorage.PutAirports(airports);
+
+            Flight flight = CreateFlight(flightRequest);
+            _flights.Add(flight);
+
+            return flight;
+        }
+
+        private static Flight CreateFlight(AddFlightRequest flightRequest)
+        {
+            int id = _id++;
+            Flight flight = new Flight
+            {
+                Id = id,
+                ArrivalTime = flightRequest.ArrivalTime,
+                Carrier = flightRequest.Carrier,
+                DepartureTime = flightRequest.DepartureTime,
+                From = flightRequest.From,
+                To = flightRequest.To
+            };
+
+            return flight;
+        }
+
+        public static bool IsSameFlight(AddFlightRequest flightRequest)
+        {
+            Flight result = _flights.FirstOrDefault(f => f.ArrivalTime == flightRequest.ArrivalTime 
+               && f.Carrier == flightRequest.Carrier
+               && f.DepartureTime == flightRequest.DepartureTime
+               && f.From.AirportCode == flightRequest.From.AirportCode
+               && f.From.City == flightRequest.From.City
+               && f.From.Country == flightRequest.From.Country
+               && f.To.AirportCode == flightRequest.To.AirportCode
+               && f.To.City == flightRequest.To.City
+               && f.To.Country == flightRequest.To.Country);
+
+            return result != null;
+        }
+
+        public static bool IsValidFlight(AddFlightRequest flightRequest)
+        {
+            //check for null and empty values
+            bool result = String.IsNullOrEmpty(flightRequest.ArrivalTime)
```

Comment:
uz kādu jautājumu result atbild?

Reply:
šajā gadījumā - is NullOrEmpty. Pārsaukt?